assets/node.yaml

```diff
@@ -18,8 +18,13 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       serviceAccount: manila-csi-driver-node-sa
       priorityClassName: system-node-critical
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
```

Comment:
I don't think we want to schedule this only on master nodes. CSI node service should be scheduled on all nodes, you can check EBS operator for an example.

Reply:
oops yeah. good catch.  node should not have a selector.

Reply:
I've removed the nodeSelector from here, and also dropped the additional toleration, as this seemed redundant.